### DISCUSSION TITLE
I'm unable to conduct a Pull request from my Earth-Analytics branch to the Main branch

### DISCUSSION THREAD

**Discussion by misterskye** [earthlab-education/Earth-Analytics-2023-01-Intro#360] [! GitHub help]
I'm getting an "X Can't Automatically Merge" warning.  Because of this I'm not seeing the folder that I should be putting the Flood Time-Series project.

**Comment by eculler** [comment]
Hi @misterskye - this warning doesn't mean that you definitely can't merge! It means that you'll have to do a bit of manual work to resolve conflicts. You can click on the 'resolve conflicts' button, and then edit files with conflicts.

You can find some instructions from GitHub on how to do this [in their conflict documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github). In general, you will want to keep all the changes from the main branch, which should be labeled in the file. If you have made changes that are not in the main branch and are marked as conflicts, you can go ahead and leave those changes in.
